src/main/java/org/openjdk/jextract/impl/ClassSourceBuilder.java

```diff
@@ -115,27 +110,13 @@ final void classBegin() {
     }
 
     final void classEnd() {
-        indent();
-        append("}\n\n");
-    }
-
-    // Internal generation helpers (used by other builders)
-
-    final void append(String s) {
-        sb.append(s);
-    }
-
-    final void append(char c) {
-        sb.append(c);
-    }
+        appendIndentedLines("""
```

Comment:
Switching to `appendIndentedLines` here so that `indent` and `append` can be removed.